src/components/MoneyRequestConfirmationList.js

```diff
@@ -273,6 +277,8 @@ function MoneyRequestConfirmationList(props) {
         );
     }, [confirm, props.selectedParticipants, props.bankAccountRoute, props.iouCurrencyCode, props.iouType, props.isReadOnly, props.policyID, selectedParticipants, splitOrRequestOptions]);
 
+    const getFileImage = () => <Lounge style={styles.moneyRequestImage} />;
```

Comment:
Not sure why we made this a function ... I think we can just inline this JSX below:

```js
{props.receiptPath && !props.isImage && <Lounge style={styles.moneyRequestImage} />}
```

Also it looks like the way we've imported `Lounge` is wrong here, but I think you might be updating that?

Reply:
Yep, updated